database/sqlite/sqlite.go

```diff
@@ -256,6 +256,11 @@ func createIndexes(c *client) error {
 		return fmt.Errorf("unable to create builds_status index for the %s table: %v", constants.TableBuild, err)
 	}
 
+	err = c.Sqlite.Exec(ddl.CreateBuildCreatedIndex).Error
```

Comment:
missing comment before index creation.

additionally, i noticed between here and the postgres addition that the index is inserted in different spots. in the postgres version it was inserted at the end of the indexes and here after the other index on the build table. if i had to pick, i like the approach here better, but feel like it should be consistent between the two no matter where it happens.